docs/general/builtins/scalar-functions.rst

```diff
@@ -2718,6 +2718,58 @@ function supports arrays of any of the :ref:`primitive types
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-array_position:
+
+``array_position(anycompatiblearray, anycompatible [, integer ] ) → integer``
+-----------------------------------------------------------------------------
+
+The ``array_position`` function returns the subscript of the first
+occurrence of the second argument in the ``array``, or ``NULL`` if it's not
+present. If the third argument is given, the search begins at that subscript.
+The ``array`` must be one-dimensional. Comparisons are done using
+``IS NOT DISTINCT FROM`` semantics, so it is possible to search for ``NULL``.
+
+::
+
+    cr> CREATE TABLE metrics(
+            ts timestamp,
+            values_sensor1 ARRAY(INTEGER),
+            values_sensor2 ARRAY(INTEGER)
+        );
+    CREATE OK, 1 row affected  (... sec)
+
+::
+
+    cr> INSERT INTO metrics VALUES (NOW(), [1, 3, 7, 4], [9, 8, 0, 2]);
+    INSERT OK, 1 row affected  (... sec)
+
+::
+
+    --Find out if a value is contained in an array
+
+    cr> SELECT array_position(values_sensor1, 10) IS NULL AS is_null
+        FROM metrics;
+    +---------+
+    | is_null |
+    +---------+
+    | TRUE    |
+    +---------+
+    SELECT 1 row in set (... sec)
```

Comment:
As commented here https://github.com/crate/crate/issues/11943#issuecomment-983549996, using `ANY` is much more efficient for existence checks of array elements. Thus we should either include a similar comment to explain that `ANY` is a more performant alternative, or we may just remove this example here.

Reply:
Added the ``TIP`` about the same.